alexBot/cogs/utils.py

```diff
@@ -41,7 +41,10 @@ async def roll(self, interaction: discord.Interaction, dice: str):
                     return await interaction.response.send_message(
                         "You can't roll more than 100 dice at once!", ephemeral=True
                     )
-            except Exception:
+            except (TypeError
+                    , ValueError
+                    , discord.HTTPException
+                    , discord.InteractionResponded):
```

Comment:
not necessary to be checked for, as the interaction will have not been responded too yet


```suggestion
):
```